shop/src/main/java/ro/msg/learning/shop/service/OrderService.java

```diff
@@ -0,0 +1,107 @@
+package ro.msg.learning.shop.service;
+
+import org.hibernate.service.spi.ServiceException;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import ro.msg.learning.shop.DTO.OrderDTO;
+import ro.msg.learning.shop.DTO.OrderedProductDTO;
+import ro.msg.learning.shop.exception.LocationNotFound;
+import ro.msg.learning.shop.exception.OrderCannotBeCompleted;
+import ro.msg.learning.shop.model.*;
+import ro.msg.learning.shop.repository.*;
+import ro.msg.learning.shop.service.strategy.LocationStrategy;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+@Service
+public class OrderService {
+
+    @Autowired
+    OrderRepository orderRepository;
+    @Autowired
+    StockRepository stockRepository;
+    @Autowired
```

Comment:
Instead of autowiring all of them you can use @RequiredArgsConstructor from Lombok.